duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -313,6 +321,27 @@ private String canonQueryString()
     return Util.join(args.toArray(), "&");
   }
 
+  private String canonXDuoHeaders(){
```

Comment:
This method is not in use yet. Not sure if I should make comment in code about it or not.